### ISSUE TITLE
docs: Missing documentation for how to use assets

### ISSUE THREAD

**Issue by robinwassen** [felixrieseberg/electron-windows-store#116]
The argument `assets` to specify the assets path is mentioned in the docs - but no instructions on what files the folder expected to contain.

There is a folder called `template/assets` with four different sample electron icons named SampleAppx.*.png. But it feels counterintuitive to create an asset folder for your app and name the icons SampleAppx.

Some kind of instructions with more details on how to use assets would be a great addition. 👍

**Comment by robinwassen** [comment]
Gave it a try - it worked to copy the `template/assets` folder, modify the icons and specify the path. Still could use some docs though.

**Comment by jameshfisher** [comment]
Indeed, the template has these "SampleAppx" filenames hard-coded ... https://github.com/felixrieseberg/electron-windows-store/blob/master/template/appxmanifest.xml#L30-L31
